cmd/swarm/access_test.go

```diff
@@ -356,11 +359,19 @@ func TestAccessPK(t *testing.T) {
 	}
 }
 
+func TestAccessACT(t *testing.T) {
+	testAccessACT(t, 0)
+}
+func TestAccessACTScale(t *testing.T) {
+	testAccessACT(t, 1000)
```

Comment:
have you tried with bigger numbers?

Reply:
hiccups start at around ~15K entries:
- it seems that one bottleneck is when creating the ACT itself. I suspect that in order to support higher amounts of grantees we'd have to do some resource pooling and multithread the ACT manifest generation
- the test also has an overhead when generating the bogus entry ECs (that alone, at 100K entries=10 seconds)